### PR TITLE
Fix installer failing to elevate

### PR DESCRIPTION
**Pull request by Nino-K**
Invoke Expression when encountered space in the paths it would through an exception nor is it the right tool for this scenario. This change removes Invoke-Expression altogether.

Signed-off-by: Nino Kodabande <nkodabande@suse.com>

### CODE REVIEW

**Comment by Nino-K** [comment]
https://github.com/rancher-sandbox/rancher-desktop/issues/2677

**Review by mook-as — APPROVED** [review]

File: build/uninstall-privileged-service.ps1 [hunk 1]
```diff
@@ -1,15 +1,16 @@
-# .SYNOPSIS
+ # .SYNOPSIS
```
Comment (mook-as) [inline build/uninstall-privileged-service.ps1:1]: ?

**Review by mook-as — COMMENTED** [review]
Basically looks good, will try running it now.

**Review by mook-as — APPROVED** [review]
Looks better than before, at least.

Note that the service isn't being uninstalled correctly; I suspect the file on disk has been deleted before the script runs.

**Comment by mook-as** [comment]
Looking at the [relevant code](https://github.com/electron-userland/electron-builder/blob/v29.30.0/packages/app-builder-lib/templates/nsis/uninstaller.nsh#L56-L61):
```nsis
  # delete the installed files
  !ifmacrodef customRemoveFiles
    !insertmacro customRemoveFiles
  !else
    RMDir /r $INSTDIR
  !endif
…
  !ifmacrodef customUnInstall
    !insertmacro customUnInstall
  !endif
```
So `customUnInstall` is executed way after the files are deleted.  Looking at the code, we may need to do the service removal in `unregisterFileAssociations` instead?  But it looks like the file has changed on `master`, so `unregisterFileAssociations` is called _after_ file deletion again.

**Review by mook-as — APPROVED** [review]

**Comment by Nino-K** [comment]
> So `customUnInstall` is executed way after the files are deleted. Looking at the code, we may need to do the service removal in `unregisterFileAssociations` instead? But it looks like the file has changed on `master`, so `unregisterFileAssociations` is called _after_ file deletion again.

I did realize it's a race that the files are being removed. One solution I can think of is to keep a copy of the service in an alternative location and used that during the uninstall and remove that location too.

**Pull request merged**